calico-cloud/image-assurance/exceptions.mdx

```diff
@@ -0,0 +1,41 @@
+---
+description: Exclude vulnerabilities from image scan assessment.
+---
+
+# Exclude vulnerabilities from image scan assessments
+
+You can exclude specific vulnerabilities from the image scan assessment (for example, a false positive).
+It can be excluded for an image version, all images in the same repository, or all images in the system.
+
+:::caution
+Excluding vulnerabilities from image assessment can significantly change how Image Assurance determines image health and can create actions you need to take to stabilize your workflow.
+- **Maximum CVSS score** - an image’s maximum CVSS score may be reduced to a lower score. An exception could eliminate the vulnerability with the highest CVSS score for an image.
+- **Scan results** - scan results for Pass/Warn/Fail could change because they are based on the maximum CVSS score. For example, Fail could change to Warn, and Warn could change to Pass.
+- **Vulnerability global alerts** - alerts may no longer be triggered. Alerts are triggered based on scan results or maximum CVSS score values of images.
+- **Admission controller policies** - pods could be created where they were previously blocked. Admission controller policies are based on vulnerability information (scan result and/or maximum CVSS score).
+:::
+
+## Exclude a single vulnerability
+
+To create a vulnerability exception:
+1. Go to **Image Assurance**, **All Scanned Images**.
+2. Select an image from the list.
+3. On the panel, open the Package, and click **Actions**, **Add Exception**.
+3. Enter the required information and click **Save**. The exception is immediately applied to your scan results.
+
+To view the list of vulnerability exceptions, go to the left navbar: **Image Assurance**, **Vulnerability Exceptions**.
+
+### Upload a CSV with many exclusions
+
+A CSV file can be uploaded to create many exceptions.
+
+The CSV must have the headers defined below on line 1. The CSV can have additional headers and they will be ignored.
+
+| Header | Explanation |
+|--------|-------------|
+| Vulnerability | The is the name of the CVE (CVE-2021-44228). |
+| Repository | This is the name of a repository and image, or a regular expression that matches many images. |
+| Justification | This is why this particular exception was created. |
```

Comment:
@davido-tigera is this just a placeholder for an eventual screenshot of the UI? Or maybe not, I don't know, the ASCII table is confusing me a bit. 

If this is supposed to give a sample of the CSV file contents, would it not be something like: 
```
Vulnerability, Repository, Justification
CVE-2021-44228, registrypath/reponame:imagetag, This is why this particular exception was created.
```

Something like that?